<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCustomPlot Scrollbar Axis Range Control Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QSpinBox" name="spinBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QSlider" name="h_Slider">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>21</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>21</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>h_Slider</zorder>
   <zorder>plot</zorder>
   <zorder>verticalScrollBar</zorder>
   <zorder>horizontalScrollBar</zorder>
   <zorder>spinBox</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSpectr">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSpectr"/>
    <addaction name="actionStatis"/>
    <addaction name="actionCos_sin"/>
    <addaction name="actionConstilation"/>
   </widget>
   <addaction name="menuSpectr"/>
  </widget>
  <action name="actionInsert_Plot">
   <property name="text">
    <string>Insert Plot</string>
   </property>
  </action>
  <action name="actionSave_Document">
   <property name="text">
    <string>Save Document...</string>
   </property>
  </action>
  <action name="actionSpectr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spectr</string>
   </property>
  </action>
  <action name="actionStatis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Statis</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OpenFile</string>
   </property>
  </action>
  <action name="actionCos_sin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Re_Im</string>
   </property>
  </action>
  <action name="actionConstilation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Constilation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
